.github/workflows/deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: "Build"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+          cache: npm
+
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: eu-west-1
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+
+      - run: npm ci
+      - run: npm run build
+      - run: |
+          mkdir about
+          mv out/* about
+          mv about out
```

Comment:
Am I understanding this correctly that we

1. Create an about folder
2. Move the contents of `out/` to `about/`
3. Move about into `out/`
4. The final folder structure is `out/about/out`

Reply:
I think it's doing 3 steps:

1:
```
out/
  index.html
```

2:
```
out/
  #nothing in here now
about/
  index.html
```

3:
```
out/
  about/
    index.html
```

The important thing is that we need to upload a directory called "about" to S3, which includes the built application. May be a simpler way to do this, I'll play around!

Reply:
I've updated the next config to output directly to `out/about` so we don't have to move files around